tools/test.py

```diff
@@ -64,8 +64,9 @@ def run_tests(tests, bin_path, test_args=[], algorithms=[], coverage=False):
                 if coverage:
                     generate_report(test_name, my_env)
 
+    os.chdir(dir_backup)
```

Comment:
What's the idea here? Does this change do anything?

Reply:
[This](https://github.com/cryspen/hacl-packages/pull/438/files#diff-e48b94354bc9ed31688c515289d6282a76f6d93dddf9fd699f6f1d9852ca8da5R478) would fail otherwise, if the directory doesn't go back to the original one the logic that checks for whether artifacts were built would fail.